providers/apache/kafka/src/airflow/providers/apache/kafka/hooks/base.py

```diff
@@ -22,6 +22,7 @@
 from confluent_kafka.admin import AdminClient
 
 from airflow.hooks.base import BaseHook
+from airflow.providers.google.cloud.hooks.managed_kafka import ManagedKafkaHook
```

Comment:
Why do we import Google hook in kafka provider?
This doesn't feel right.

Reply:
KafkaBaseHook is used in many places this import force to install google provider to use the Kafka provider.
Am I missing something?

Reply:
Ah indeed. sorry I missed that one. Good you noticed it.

Reply:
@eladkal @potiuk I am ready to prepare a fix ASAP. But I have a question. In case when we can't use a google Hook inside Kafka provider what is a better way to use code from the google provider? Could I make a conditional import here as a solution? 

The reason for using Google Hook was that for the Kafka cluster managed by Google for establishing connection we need to pass a function which generates a token for Confluent with google's credentials. And a function which gets a google's credential exists only inside the Google provider.